docs/manual/07-operations/01-deployment-strategies.md

```diff
@@ -688,6 +688,52 @@ migrations/
 └── 004_add_projection_checkpoints.sql
 ```
 
+#### Metadata Column Upgrade (eventcore-postgres)
+
+EventCore 0.1 introduced the `metadata` column to persist per-event envelopes. Existing
+deployments that created the `eventcore_events` table before this change can roll out the
+new column without downtime by running the additive migration `0002_add_metadata_column.sql`
+shipped with the `eventcore-postgres` crate.
```

Comment:
There are no "existing deployments" and we don't worry about backward compatibility for versions < 1.0.0